packages/esm-service-queues-app/src/create-queue-entry/queue-fields/queue-fields.component.tsx

```diff
@@ -34,6 +35,7 @@ const createQueueServiceSchema = (t: TFunction) =>
     queueLocation: z.string().trim(),
     queueService: z.string().trim(),
     priority: z.string().trim(),
+    provider: z.string().optional(),
```

Comment:
Did you intentionally leave out the trimming?

Reply:
Yes , to remove trailing spaces